static/app/views/dashboardsV2/widgetCard.tsx

```diff
@@ -81,6 +83,11 @@ class WidgetCard extends React.Component<Props> {
 
     return (
       <ToolbarPanel>
+        <Feature features={['organizations:dashboard-widget-resizing']}>
+          <SizeSelectContainer style={{visibility: hideToolbar ? 'hidden' : 'visible'}}>
```

Comment:
Noticed that my size selector wasn't being hidden on drag. I duplicated the visibility styling here to keep my changes low impact to the surrounding code, should I have made a container that surrounds both `Feature` and `IconContainer` instead for DRY-ness? 🤔